framework/src/loops/ComputeResidualAndJacobianThread.C

```diff
@@ -90,10 +90,11 @@ ComputeResidualAndJacobianThread::join(const ComputeResidualAndJacobianThread &
 void
 ComputeResidualAndJacobianThread::determineObjectWarehouses()
 {
-  if (_vector_tags.size() &&
-      _vector_tags.size() != _fe_problem.numVectorTags(Moose::VECTOR_TAG_RESIDUAL))
-    mooseError("Can only currently compute the residual and Jacobian together if we are computing "
-               "the full suite of residual tags");
+  // if (_vector_tags.size() &&
+  //     _vector_tags.size() != _fe_problem.numVectorTags(Moose::VECTOR_TAG_RESIDUAL))
+  //   mooseError("Can only currently compute the residual and Jacobian together if we are computing
+  //   "
+  //              "the full suite of residual tags");
```

Comment:
so we work with the full suite of jacobian tags but not residual tags?

Reply:
Yeah I need to work on this a bit! Potentially in a separate, preceding PR.